garak/analyze/deep_analysis/analytics.py

```diff
@@ -0,0 +1,264 @@
+        "promptinject.HijackHateHumansMini",
+        "promptinject.HijackKillHumans",
+        "promptinject.HijackKillHumansMini",
+        "promptinject.HijackLongPrompt",
+        "promptinject.HijackLongPromptMini",
+        "suffix.BEAST",
+        "suffix.GCG",
+        "suffix.GCGCached",
+        "tap.PAIR",
+        "tap.TAP",
+        "tap.TAPCached",
+        "visual_jailbreak.FigStep",
+        "visual_jailbreak.FigStepTiny",
+    ],
+    "system": [],
+    "tokenizer": [],
+    "code": [
+        "malwaregen.Evasion",
+        "malwaregen.Payload",
+        "malwaregen.SubFunctions",
+        "malwaregen.TopLevel",
+    ],
+    "misleading": [
+        "goodside.WhoIsRiley",
+        "misleading.FalseAssertion50",
+        "snowball.GraphConnectivity",
+        "snowball.GraphConnectivityMini",
+    ],
+    "training_data": [
+        "grandma.Win10",
+        "grandma.Win11",
+        "leakreplay.GuardianCloze",
+        "leakreplay.LiteratureCloze",
+        "leakreplay.LiteratureCloze80",
+        "leakreplay.NYTCloze",
+        "leakreplay.PotterCloze",
+        "divergence.Repeat",
+        "divergence.RepeatExtended",
+    ],
+    "harm": [
+        "atkgen.Tox",
+        "continuation.ContinueSlursReclaimedSlurs",
+        "continuation.ContinueSlursReclaimedSlursMini",
+        "lmrc.Bullying",
+        "lmrc.Deadnaming",
+        "lmrc.QuackMedicine",
+        "lmrc.SexualContent",
+        "lmrc.Sexualisation",
+        "lmrc.SlurUsage",
+        "topic.WordnetControversial",
+    ],
+    "bypass": [
+        "encoding.InjectAscii85",
+        "encoding.InjectBase16",
+        "encoding.InjectBase2048",
+        "encoding.InjectBase32",
+        "encoding.InjectBraille",
+        "encoding.InjectEcoji",
+        "encoding.InjectHex",
+        "encoding.InjectMime",
+        "encoding.InjectMorse",
+        "encoding.InjectNato",
+        "encoding.InjectROT13",
+        "encoding.InjectZalgo",
+    ],
+}
+
+TIER_2_PROBES = list(set().union(TIER_2_PROBE_GROUPS.values()))
+
+CALIBRATION_DATA = data_path / "calibration" / "calibration.json"
+
+JAILBREAK_FEEDBACK_LOCATION = data_path / "deep_analysis" / "jailbreak.json"
+
+
+@lru_cache
+def load_calibration_data(calibration_filename=CALIBRATION_DATA):
+    with open(calibration_filename, "r", encoding="utf-8") as f:
+        data = json.loads(f.read().strip())
+
+    probe_data = {key.split("/")[0]: value for key, value in data.items()}
+    aggregate_scores = dict()
+    for k, v in TIER_1_PROBE_GROUPS.items():
+        scores = [probe_data[probe]["mu"] for probe in v if probe in probe_data.keys()]
+        minus_sigma_scores = [
+            probe_data[probe]["mu"] - probe_data[probe]["sigma"]
+            for probe in v
+            if probe in probe_data.keys()
+        ]
+        plus_sigma_scores = [
+            probe_data[probe]["mu"] + probe_data[probe]["sigma"]
+            for probe in v
+            if probe in probe_data.keys()
+        ]
```

Comment:
what do these represent? the ±1 s.d. bounds of an average model's score for a given probe:detector pair?